scripts/script_utilities.js

```diff
@@ -0,0 +1,11 @@
+
+const getArgumentsHelper = function() {
+  const arguments = process.argv.slice(4)
+  const index = arguments.indexOf("--network")
+  if (index > -1) {
+    arguments.splice(index, 2)
```

Comment:
Is this supposed to be "splice" or "slice"?

Reply:
splice